### PR TITLE
[MM-60823] Replace FormattedMarkdownMessage in 'webapp/channels/src/components/invitation_modal/add_to_channels.tsx' with FormattedMessage

### PR DESCRIPTION
**Pull request by gvarma28**
#### Summary
Replace FormattedMarkdownMessage in `webapp/channels/src/components/invitation_modal/add_to_channels.tsx` with FormattedMessage

#### Ticket Link
Fixes: #28503 

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Can you update the test snapshots?

**Comment by gvarma28** [comment]
I had missed adding the `values` field. This should be good now.

File: webapp/channels/src/components/invitation_modal/add_to_channels.tsx [hunk 1]
```diff
@@ -88,14 +87,20 @@ export default function AddToChannels(props: Props) {
             {' '}
             <span className='InviteView__sectionTitleParenthetical'>
                 {(props.channelToInvite && props.inviteType === InviteType.MEMBER) ? (
-                    <FormattedMarkdownMessage
-                        id='invite_modal.add_channels_title_c'
-                        defaultMessage='**(Optional)**'
+                    <FormattedMessage
+                        id='invite_modal.add_channels_title_member'
+                        defaultMessage='<b>(Optional)</b>'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/invitation_modal/add_to_channels.tsx:92]: can we change the casing to `optional` in align with `required`?

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]

**Comment by gvarma28** [comment]
@M-ZubairAhmed 
Added the suggested change. 🚀

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Review by pvev — APPROVED** [review]
LGTM! thanks @gvarma28 🎉

**Pull request merged**